### ISSUE TITLE
refund Gratipay 1.0 balances

### ISSUE THREAD

**Issue by whit537** [gratipay/gratipay.com#3539]
In [Gratipay 2.0](https://medium.com/gratipay-blog/gratipay-2-0-2453d3c53077) we state the following:

> You have until August 19, 2015 (90 days) to migrate your incoming payments from your Gratipay user account to a new Gratipay Team; this will also pay out your current balance. On that date, we intend to clear all remaining payments to Gratipay users, and to clear the balance of any user that is not an owner or member of a team, by refunding the money back to the sources it originally came from.

Since that was published, we've added a way to [cash out 1.0 balances](https://github.com/gratipay/gratipay.com/issues/3481), so that's another option folks have before the big refund. We've also pushed the date back to October 1.

#### Notify
- [ ] https://gratipay.freshdesk.com/helpdesk/tickets/3103

**Comment by whit537** [comment]
We should notify affected people about this 30 days out, on Monday, July 20.

**Comment by whit537** [comment]
##### Draft 1

> You are receiving this message because you are carrying a balance of $`__`.`__` on Gratipay.
>
> Three months ago, prompted by legal concerns with our old terms of service, we wrote new terms of service and relaunched as Gratipay 2.0. As mentioned in [the 2.0 announcement](https://medium.com/gratipay-blog/gratipay-2-0-2453d3c53077), we intend to clear out remaining Gratipay 1.0 balances by sending funds back to the original donors to the extent possible. The date for this will be August 20, thirty days from today.
>
If you would like to withdraw your Gratipay 1.0 balance rather than having it refunded to your original donors, then you have two options:
> 
1. Apply for [a new Gratipay Team]().
2. Apply for [a one-time payout]().
>
In either case, if we approve your application, we will pay out your balance during the next weekly payment cycle. There are five more payment cycles from today before we refund remaining balances. Whatever we are unable to refund to the original donor, we will donate to Watsi.

**Comment by whit537** [comment]
August 19 is a Wednesday. Let's push back a day to August 20, to give people that extra payment cycle to get in on payouts.

**Comment by whit537** [comment]
That would put 30 days out at Tuesday, July 21. We should send a second email a week out, on August 13. And of course we should send a final email when it's done.

**Comment by whit537** [comment]
Here's what we should get done before we send out the 30 day email:

 - Make team applications suck less.
 - Implement a one-time payout application.

**Comment by whit537** [comment]
Should we try and pull of #3378 as part of this?

**Comment by whit537** [comment]
We should also make August 20 the deadline to migrate tips to a new team.

**Comment by whit537** [comment]
A precedent, from two weeks after we launched:

> As a result, most of the money we collected—over 90%—has gone unclaimed. As of today there’s nearly $150,000 in earmarked money sitting in a separate, untouched bank account.

http://blog.readability.com/2012/06/announcement/

**Comment by whit537** [comment]
First draft: "[Flushing the Lines](https://medium.com/@Gratipay/97d54e6d40ed)."

**Comment by whit537** [comment]
@mattbk Can you read that over and see if it makes sense from a support point of view? What questions are people going to ask? Does this post give you the info you need to be able to answer their questions?

Also, I think we should push back the date on this. We need to get some stuff done before we can make an announcement:

 - Clean up more [Pivot](https://github.com/gratipay/gratipay.com/milestones/Pivot) bugs, ideally all of them. We want lots of traffic to Gratipay with this post, and we want to not confuse people with bugs when they show up.
 - Implement the application process for Option 2 (should include email notifications as appropriate), and finish implementing the process for Option 3, which now is supposed to *include* Option 2 as well as #3568. We want @mattbk to be able to manage the review queue for Option 2.
 - Pin down the escrow work we want to get done and budget enough time for it. I want to come out of this with a reviewed (https://github.com/gratipay/inside.gratipay.com/issues/285), double-entry-accounted-for (#3618) escrow, with minimal escrow sources (#1383).
 - Check on https://github.com/gratipay/legal/issues/10 about donating remainder to charity, and review regulatory references.

Let's do this right, people.

[![move-slow-and-make-things](https://cloud.githubusercontent.com/assets/134455/8788792/de40a69c-2f0a-11e5-8f95-1431719787a7.png)](https://www.stickermule.com/marketplace/1580-move-slow-and-make-things)

**Comment by mattbk** [comment]
Questions people may have on first reading:
- Flushing money?  That sounds scary.
- What's a tip graph?
- Takes until end of 2nd paragraph to know where the money is going (back to donors or to charity).
- That "subscriptions" word pops up again.
- I still don't know what a tip graph is, and why that makes it impossible for me to apply for a team.
- "refund your balance back upstream to the donors it originally came from" You're taking my money and giving it back to the original donors?

Most people are interested in one thing: "What's happening to my money?"  If they have to wade through a lot of details, they'll skip reading everything and end up confused.  I suggest you keep the first paragraph (or variation), then skip to "Your Options" section, then include the rest of the first section in details somewhere.  This will let people get the information they need up-front, but also be able to dig into details and motivations if they really want to.

**Comment by stevepiercy** [comment]
Option 3 does not resolve an issue that I reported in [FreshDesk ticket #2427](https://gratipay.freshdesk.com/support/tickets/2427). Specifically, that I don't want to create a new (and second) PayPal account for a Team when I already have one for my current ~user. I don't want to do create a new PayPal account for KnopProject (e.g., paypal@knop-project.org) because it would require putting my personal information into an organization's PayPal account, and PayPal can and does freeze organization assets.

**Comment by mattbk** [comment]
I think the confusion here lies in the fact that you have a ~user account set up to be the owner of your team, when ideally you would just use your own ~user account as the owner.  That way all payouts go to the owner (~user).  

This is predicated on the plan to [bring back payroll](https://github.com/gratipay/gratipay.com/issues/3433), wherein a team would be made up of multiple ~users (one of them being owner), and all divying up of funds would happen within Gratipay.

**Comment by stevepiercy** [comment]
@mattbk nailed it.

For me, as the owner of a Team, bringing back payroll is a requirement to use Gratipay.

I'm not certain about the current transition state, but I'm patient and will wait for good things to come.

**Comment by mattbk** [comment]
I already sent this to @stevepiercy, but copying this here from https://github.com/gratipay/inside.gratipay.com/issues/242#issuecomment-123808420 for clarification:
> > Does a ~user/owner assume liability for reporting income to the team to the government, in the same way an employer must?

> Yes. That's section 5.iii of the new [terms](https://gratipay.com/about/policies/terms-of-service).

> You are responsible for paying any taxes and withholdings associated with your use of the Service. Team Owners, and not Gratipay, are responsible for compliance with all laws, as well as any taxes and withholding requirements, applicable to payments made to Team Members (whether through the Payroll feature or otherwise), including without limitation payroll tax, unemployment insurance, and worker compensation insurance payments. Team Owners are also responsible for providing any required tax and other documentation to employees, independent contractors, tax authorities, or any other person.

**Comment by stevepiercy** [comment]
@whit537 For the purposes of this draft, and for those who wish to become Team Owners, I think it is important to emphasize section 5.iii in the new terms. Assuming the role of Team Owner has greater responsibility and more duties than an ordinary User, and constitutes a significant change from the ordinary User perspective. Currently the draft has no mention of tax or other additional responsibility for this role. Thanks for your consideration.

**Comment by mattbk** [comment]
Per https://github.com/gratipay/gratipay.com/pull/3510#issuecomment-124223679, I think the team requirement should be very obvious if an email goes out about flushing or announcing this post.

**Comment by whit537** [comment]
Let's handle the refunding of escrow separately from the deletion of the tip graph. We can delete the tip graph after Balanced is gone, but we need to refund escrow before then. Let's keep this as simple as possible.

**Comment by whit537** [comment]
Hmm ... it'd be really nice to make a clean break, though ...

**Comment by whit537** [comment]
> I'm patient and will wait for good things to come.

You're awesome, @stevepiercy. :-)

**Comment by whit537** [comment]
Second draft, incorporating feedback from @mattbk and https://github.com/gratipay/legal/issues/10:

"[Refunding 1.0](https://medium.com/@Gratipay/97d54e6d40ed)"

**Comment by mattbk** [comment]
"Option 3 automatically includes Option 2."  should be "2 includes 1."

On July 29, 2015 4:02:47 PM CDT, Chad Whitacre <notifications@github.com> wrote:
>Second draft, incorporating feedback from @mattbk and
>https://github.com/gratipay/legal/issues/10:
>
>"[Refunding 1.0](https://medium.com/@Gratipay/97d54e6d40ed)"
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/gratipay/gratipay.com/issues/3539#issuecomment-126094939

-- 
Sent from a phone that, although intelligent, is not street smart.

**Comment by whit537** [comment]
Good catch, fixed.

**Comment by whit537** [comment]
I'm waffling on whether or not to delete the tip graph as part of this. Currently leaning towards yes, delete the tip graph, for a clean break with 1.0. We don't want to have to make a second communications push on a similar topic a month later.

**Comment by whit537** [comment]
Delete the tip graph means:

```
drop table tips;
```

**Comment by whit537** [comment]
`tips` is the table where we recorded the intentions to weekly tip `~users`.

**Comment by whit537** [comment]
"[Refunding 1.0](https://medium.com/@Gratipay/97d54e6d40ed)" assumes charging in arrears (#3378) is already done.

**Comment by mattbk** [comment]
It would be interesting to someday bring back the tip data for people to see, but maybe the schism between 1.0 and 2.0 is too great.  That doesn't mess with people's history at all (useful for personal accounting and taxes), does it?

**Comment by whit537** [comment]
@mattbk Maybe an "export tips" feature? Reticketed as #3651.

**Comment by whit537** [comment]
And no, that doesn't mess with history. History comes from `exchanges`, `transfers`, and (#3455) `payments`. After #3618 history will be a journal of the ledger.

**Comment by whit537** [comment]
To: Balanced Support
Subject: Gratipay plans

> Hey Balanced!
>
Not sure who's left there on the skeleton crew, but I wanted to get in touch to let you know about our plans over here at Gratipay for closing out our Balanced account. We're planning to really take it down to the wire, and are hoping to do a lot of refunds during the first week of October.
>
The reason is that we have about $150,000 in "Gratipay 1.0 money" that we need to flush out of our system now that we have relaunched as Gratipay 2.0. We're trying to pay out as much of that as possible, but the bottom line is that we're expecting to have to refund most of it, and I understand that October 9 is the planned last day to be able to do that.
>
I wanted to get on your radar to let you know what our plans are. We're aiming for October 1 for our [Big 1.0 Refund](https://github.com/gratipay/gratipay.com/issues/3539). Are you still on track to keep the refund API open through October 9? Any red flags you see with our plan?

**Comment by whit537** [comment]
We are really taking this down to the gosh darn wire. Today is September 1. As I revisit the [blog post](https://github.com/gratipay/team-review/issues) in order to publish it tonight, I'm putting the timeline together: we need to give people 30 days to apply for a one-time payout or a Team. **But**, we then need to give ourselves **time to review them all.** We need at least a week for review, which pushes the big refund back to October 8 ... the day before Balanced turns off the refund API! :fearful:

**Comment by whit537** [comment]
I haven't heard back from Balanced yet. We should really make contact with them. I used to have Matin's number but I lost it when my phone died. I'll send him an email ...

To: Matin
Subject: big refund on October 8?

> Greetings, friend. Hope you're doing well and that Balanced is coming in for a smooth landing. :)
>
I'm writing because over here in Gratipay-land we're faced with the need to refund most of the $140,000 we've accumulated in escrow. Our ideal timeframe for doing this is early October. Since we're taking this so close to the wire, I wanted to coordinate with you to make sure our plan will to work.
>
Could you give me a call at 412-925-4220? I lost all my contacts or I'd call you. :)
>
Thanks, Matin!

**Comment by whit537** [comment]
> I'm waffling on whether or not to delete the tip graph as part of this. Currently leaning towards yes, delete the tip graph, for a clean break with 1.0. We don't want to have to make a second communications push on a similar topic a month later.

I'm still thinking "yes," for the reason articulated.

**Comment by whit537** [comment]
Alright, I revised the blog post. It's ready for final review:

https://medium.com/@Gratipay/97d54e6d40ed

We can publish with Team review as it stands today (though we'll at least need to add an "Apply" button to the homepage), but we can't publish without the one-time payout application (#3743). I'm going to work on that next.

We should also advertise this blog post somewhere on the site, probably on every page. Speaking of advertising, how else should we try to get the word out? Do we need to go back to Twitter for this? Thoughts, @mattbk et al.?

**Comment by whit537** [comment]
Published as unlisted, so I can get the link to include in #3743:

https://medium.com/gratipay-blog/140-000-97d54e6d40ed

**Comment by whit537** [comment]
I'm not going to get #3743 done tonight. I'm too spent. :-(

I propose to make a simple "1.0 Payout" placeholder page that tells people to send us email. Hopefully I can #3743 done tomorrow morning.

**Comment by mattbk** [comment]
I know this will add work for review, but I think a welcoming statement like "If you think your team might clash with the Gratipay brand, you're still invited to apply" or something that sounds more inviting. 

On September 1, 2015 7:40:55 PM CDT, Chad Whitacre <notifications@github.com> wrote:
>Published as unlisted, so I can get the link to include in #3743:
>
>https://medium.com/gratipay-blog/140-000-97d54e6d40ed
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/gratipay/gratipay.com/issues/3539#issuecomment-136903306

-- 
Sent from a phone that, although intelligent, is not street smart.

**Comment by whit537** [comment]
@mattbk Fair enough. How about simply linking out to the howto without going into further details in the blog post about the specifics of review?

>If you’d like to continue receiving money on Gratipay, then you should [apply for a Team](https://gratipay.com/new), and we’ll [review](http://inside.gratipay.com/howto/review-teams) your application.

**Comment by whit537** [comment]
1.0 payout stub (#3745) is deployed, and [blog post](https://medium.com/gratipay-blog/140-000-97d54e6d40ed) is fully published. Happy to keep tweaking it ...

**Comment by stevepiercy** [comment]
Looks good. Two suggestions.

* How do I know whether I have funds in my Gratipay account waiting for me to withdraw? (Yes, a dumb question, but... :wink: )
* Add a "Do nothing" option, with a link to check a user's Gratipay balance.

**Comment by whit537** [comment]
Good ideas, @stevepiercy. #3744 will answer the "Do I have a balance?" question. I like the idea of adding a "Do nothing" option on the blog post, I think it can link to the same place if we word the page properly.

**Comment by mattbk** [comment]
> How about simply linking out to the howto without going into further details in the blog post about the specifics of review?

Looks good.

**Comment by whit537** [comment]
We have a number of participants with a 1.0 balance less than PayPal's minimum payout of $0.50. How are we going to pay them out? Dwolla (#726)?

**Comment by whit537** [comment]
The total is less than $50. That's small enough that we could probably get away with donating it somewhere.

**Comment by whit537** [comment]
Well, I guess we refund it! :)

**Comment by whit537** [comment]
#### Balances by Range

| Range ($) | Number |
| ---------- | ------------- |
| 1,000.00+ | 17 |
| 100.00—999.99 | 271 |
| 10.00—99.99 | 812 |
| 0.50—9.99 | 937 |
| 0.01—0.49 | 222 |
| *total* | 2,259 |

**Comment by whit537** [comment]
50 participants received under Gratipay 1.0, and have a balance now, but had a zero balance at the Gratipocalypse.

**Comment by whit537** [comment]
Scratch that: only 7 (I wasn't taking exchange status into account).

**Comment by whit537** [comment]
From Matin:

> Sorry for the slow response.
>
I'll give you a call tomorrow to figure out how best to coordinate this.

----------------

> Thanks, Matin. :-)

**Comment by whit537** [comment]
Coordinating with PayPal reticketed as https://github.com/gratipay/inside.gratipay.com/issues/330.

**Comment by whit537** [comment]
The 1.0 payout status page is up! #3743

I've made the following modifications to the [blog post](https://medium.com/gratipay-blog/140-000-97d54e6d40ed):

Changed title from "$140,000" to "$130,000."

Changed first sentence from:

> On October 8, 2015, we plan to refund almost all of the $140,000 that we’re currently holding in escrow for about 4,700 of you.

to (see numbers at https://github.com/gratipay/gratipay.com/issues/3743#issuecomment-138388374):

> On October 8, 2015, we plan to refund about $130,000 that we’re currently holding in escrow for about 2,000 of you — unless you withdraw it first.

Added "Do nothing" option.

**Comment by whit537** [comment]
```sql
=> select status_of_1_0_payout, count(*), sum(balance) from participants
group by status_of_1_0_payout order by sum desc;
┌──────────────────────┬────────┬───────────┐
│ status_of_1_0_payout │ count  │    sum    │
├──────────────────────┼────────┼───────────┤
│ pending-application  │   1952 │ 127447.47 │
│ completed            │ 110848 │  10272.68 │
│ pending-payout       │      5 │   1622.64 │
│ too-little           │    207 │     42.99 │
└──────────────────────┴────────┴───────────┘
(4 rows)
```

**Comment by stevepiercy** [comment]
@whit537 

Should be "three"

> You have **two** options to withdraw Gratipay 1.0 money:

> 1. Do nothing.

**Comment by whit537** [comment]
Good catch. Changed from:

> You have two options to withdraw Gratipay 1.0 money:

To:

> You have three options regarding your Gratipay 1.0 money:

**Comment by whit537** [comment]
I've shared the blog post with [The Changelog](https://github.com/thechangelog/ping/issues/277) and [Pycoder's Weekly](http://pycoders.com/).

![screen shot 2015-09-11 at 3 05 17 pm](https://cloud.githubusercontent.com/assets/134455/9823812/a3e57008-5896-11e5-9f47-43e22c8a59c8.png)

**Comment by mattbk** [comment]
What will happen to ~users who have a balance but who are unable/unwilling* to create a PayPal account before refunds happen?  Is it even possible we'd have a bank payout available before October 8th?

*Unable may be due to local laws, e.g., PayPal is not available in South Africa.

**Comment by whit537** [comment]
@mattbk There's a slim chance we could have U.S. bank payouts before Oct. 8, if someone implements Dwolla (#726), but certainly not non-U.S. If someone in South Africa needs a payout we would have to find an alternative. E.g., we used Transferwise for the first time recently: https://github.com/gratipay/inside.gratipay.com/issues/249. I think we should take this on a case-by-case basis since most people will probably be fine with PayPal. Sound right?

**Comment by mattbk** [comment]
> We have a number of participants with a 1.0 balance less than PayPal's minimum payout of $0.50. How are we going to pay them out? Dwolla (#726)?

https://gratipay.freshdesk.com/helpdesk/tickets/3103 says she doesn't want it.  The ~user is marked as "too little" but would it go to donation if I changed that to "completed"?  User also wants to close account, otherwise this wouldn't matter.

**Comment by whit537** [comment]
@mattbk Leave it as `pending-application`. That's what we'll use to determine what to refund on Oct 8. I think she'll need to wait until Oct 8 to close her account, maybe start a "Notify" list?

**Comment by mattbk** [comment]
Can't use pending-application, there's no payout route.

**Comment by mattbk** [comment]
> If someone in South Africa needs a payout we would have to find an alternative. E.g., we used Transferwise for the first time recently: gratipay/inside.gratipay.com#249. I think we should take this on a case-by-case basis since most people will probably be fine with PayPal. Sound right?

Sounds right.

**Comment by whit537** [comment]
> Can't use `pending-application`, there's no payout route.

Sorry ... `too-little` is also fine.

**Comment by whit537** [comment]
Just got off the phone with Matin. Walked him through our situation. Take-aways:

 - Balanced is still on track to have the refund API open until October 9. No problem from that perspective.
 - Whatever is left in our Balanced escrow account after October 9 will be sent to our bank account.
 - **Issuing lots of tiny, old refunds is very likely to trigger risk checks at multiple levels**: with Balanced's banking partner, with the card brands, and with the issuing banks.
 - Therefore, we need to:
  - Try to payout as much as possible, to reduce the need to refund.
  - Sort refunds according to age and size, and only issue refunds above some threshold, to be determined together based on an analysis of the money left in the system.

**Comment by whit537** [comment]
@mattbk If I make a dashboard page listing `pending-application` ~users by `balance`, would you be willing and able to reach out to them personally to encourage them to apply for a payout?

**Comment by whit537** [comment]
Matin mentioned a legal term, [escheat](https://en.wikipedia.org/wiki/Escheat), which has to do with disposing of unclaimed property, and could be relevant to how we handle the money we're left with on October 10.

**Comment by whit537** [comment]
To: Matin

> Thanks for the call, Matin. Here's my summary:
>
https://github.com/gratipay/gratipay.com/issues/3539#issuecomment-142102062
>
Feel free to subscribe to that ticket for updates if you like. I'll email you again once I have data.

**Comment by whit537** [comment]
Submitted to HN:

`https://news.ycombinator.com/item ?id=10255117`

(URL intentionally broken according to HN Referer lore).

Let's get a couple upvotes so it gets some traction?

**Comment by whit537** [comment]
We need the money in our Balanced account in order to do the refunds.

We have $28,402.62 in there right now. That's enough for the ~$10,000 in advance fees next week (#3763).

We need to decide how much we need in there for the rest—and how we're going to handle payouts once we don't have enough cash to float the PayPal transfer week to week. We should start moving money on Monday.

**Comment by whit537** [comment]
To: Matin

> Quick note to say we're behind on the refund lists, and we may not get to the refunds on Balanced at all. I'll keep you posted.

**Comment by whit537** [comment]
> We should start moving money on Monday.

We didn't start moving money on Monday. Therefore, we only have $28,402.62 in our Balanced account, and it takes 3-5 business days to add money. The most we can refund is $28,402.62, well short of the ~$130,000 we were targeting to refund before tomorrow.
